generators/app/templates/src/containers/Welcome/welcome.component.js

```diff
@@ -47,13 +47,14 @@ const WelcomePageContent = props => {
                 },
                 onError: error => {
                   if (error && error.statusText) {
-                    errorToaster(error.statusText);
+                    errorToaster(error.statusText, 'Error');
```

Comment:
Title should be translated instead of hardcoded

Reply:
Done